5.js

```diff
@@ -0,0 +1,29 @@
+var $block = document.getElementById('block');
+var $table = document.createElement('table');
+var letters = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H'];
+
+for (var i = 0; i <= 9; i++) {
+    var $row = document.createElement('tr');
+
+    for (var j = 0; j <= 9; j++) {
+        var $cell = document.createElement('td'); 
+
+        if ((i == 0) || (i == 9)) {
```

Comment:
Рекомендую использовать строгое сравнение
```suggestion
        if ((i === 0) || (i === 9)) {
```